### DISCUSSION TITLE
What is the Graph connector in the app manifest?

### DISCUSSION THREAD

**Discussion by svrooij** [microsoft/hack-together-teams#26] [Q&A] [answered]
I was checking out all the possibilities of a Teams app and found the somewhat mysterious [Graph Connector](https://learn.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema-dev-preview).

what is this and how can we use it? Maybe there should be a link to graph connectors in the Teams Manifest documentation.

**Comment by waldekmastykarz** [comment]
It's a packaging construct meant to simplify distribution of Graph Connectors alongside Teams apps. Do you have something in mind for this? 😄

**Reply by svrooij** [reply]
I was wondering where I can find documentation on Graph Connectors, that sounds interesting. My guess is that you can use it to add additional data to the graph api or something like that?

**Reply by waldekmastykarz** [reply, marked as answer]
You can read more about connectors in our docs at https://learn.microsoft.com/graph/connecting-external-content-connectors-overview
